src/components/ec-inline-input-field/ec-inline-input-field.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div
+    class="tw-truncate"
+    data-test="ec-inline-input-field"
+  >
+    <div class="tw-mini-header">{{ label }}</div>
```

Comment:
This div should be a label in editing mode and it should point to the ID of the <ec-input-field>. Otherwise we will have just an empty input without a label. Accessibility panel complains about it too.